css/style.css

```diff
@@ -1,19 +1,43 @@
 /* General page styles */
 /* We should use a mobile first approach having out breack point at 768px */
-/*Comodities */
+
+/* START GENERAL STYLES */
+*,
+*::after,
+*::before {
```

Comment:
What are you doing in this line?

Reply:
@ArleyAV Could you please answer this question?

Reply:
@hanna446 

Because this way we save code's lines. What I mean is we are applying the same rules to this selectors

Reply:
@hanna446 The * character makes reference to all elements in the dom, so this code is applying to all elements `box-sizing: border-box` which include padding and border in the element's total width and height. Also with `::before` and `::after` you are applying this to all pseudo-lements.